composer.json

```diff
@@ -1,5 +1,22 @@
 {
     "require-dev": {
-        "wp-coding-standards/wpcs": "^2.3"
+        "wp-coding-standards/wpcs": "^2.3",
+        "wpify/scoper": "^3.1"
+    },
+    "config": {
+        "allow-plugins": {
+            "wpify/scoper": true
+        }
+    },
+    "extra": {
+        "wpify-scoper": {
+            "prefix": "KCO",
```

Comment:
I would recommend using `KrokedilKCODeps` as the prefix, even if its longer its more descriptive and prevents any possible namespace conflicts as well. And its similar to how we have done it in [other plugins](https://github.com/krokedil/avarda-checkout-for-woocommerce/blob/master/composer.json#L23) as well.